reporting.md

```diff
@@ -29,70 +31,60 @@ During installation, you will be prompted to log in with a user account. The acc
 
 ### Running the installer
 - From the [Simeon portal](https://app.simeoncloud.com/), click **Install** on the navigation pane > select the **Install Power BI Reporting** tab
-- Enter the domain name of the tenant > **INSTALL** > authenticate with an account that meets the [prerequisites](#prerequisites) > when prompted, select the Azure Subscription and desired location for the SQL server
+- Enter the domain name of the tenant > **INSTALL**
+- Authenticate with an account that meets the [prerequisites](#prerequisites)
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/feature/loganalytics/assets/images/power_bi_auth_to_tenant.png" width: 338; height: auto;/>
+- When prompted, select the Azure Subscription
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/feature/loganalytics/assets/images/power_bi_subscription.png" width: 337; height: auto;/>
+- When prompted, select an Azure location
```

Comment:
I think that certain locations don't work (such as Europe). Maybe we could add a generic sentence about that scenario here, I'm not sure